guides/source/rails_on_rack.md

```diff
@@ -215,9 +215,9 @@ Much of Action Controller's functionality is implemented as Middlewares. The fol
 
 * Used to serve static files from the public directory. Disabled if `config.public_file_server.enabled` is `false`.
 
-**`Rack::Lock`**
```

Comment:
The list mirrors the output of `rails middleware` from a fresh app. I'd say it's covered in configuring.md.